kit/gcp/data-lagoon/buildingblock-backplane/documentation.tf

```diff
@@ -0,0 +1,5 @@
+output "documentation_md" {
+  value = <<EOF
+This module manages the building block backend for GCP.
+EOF
+}
```

Comment:
https://likvid-bank.github.io/likvid-cloudfoundation/platforms/azure/azure-buildingblocks-connectivity-backplane.html as an example. Maybe some infos what is or which permissions are added.